wikipedia/track.json

```diff
@@ -0,0 +1,34 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Benchmark for search with Wikipedia data",
+  "search_applications": [
+    {}
+  ],
+  "indices": [
+    {
+      "name": "wikipedia",
+      "body": "wikipedia-{{index_mapping_type | default("minimal")}}-mapping.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "wikipedia",
+      "base-url": "https://rally-tracks.elastic.co/wikipedia",
+      "documents": [
+        {
+          "source-file": "documents-1k.json.bz2",
+          "document-count": 1000,
+          "compressed-bytes": 10257016,
+          "uncompressed-bytes": 37065115
+        }
```

Comment:
I reverted this change that should not have been pushed.
I think you should have used esrally with the `--test-mode` argument instead of making this change.
This way it would have automatically downloaded the `documents-1k.json` file instead of the full.

Reply:
That makes sense.